### DISCUSSION TITLE
Prowlarr LXC - Update script reports completed/UI reports not up to date

### DISCUSSION THREAD

**Discussion by Hoosier-IT** [tteck/Proxmox#2719] [Help Needed] [answered]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

Prowlarr LXC:

The Prowlarr LXC helper script was ran inside the LXC console to process the recent update. 

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/prowlarr.sh)"

After completion, shutdown and started:

Prowlarr LXC UI /system/status

Reports: New update is available 1.15.0.4361 — Mar 27 2024

Installed shows:
Version 1.14.3.4333
.NET Yes (6.0.25)
Database Sqlite 3.40.1
Database Migration 39
AppData Directory /var/lib/prowlarr
Startup Directory /opt/Prowlarr
Mode Console
Uptime 00:03:10

No errors in processing update script but the UI still shows as not up to date.

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

![image](https://github.com/tteck/Proxmox/assets/8570961/31d64f31-05b4-4f3d-a87e-3c888ebd328d)

**Comment by Hoosier-IT** [comment]
![image](https://github.com/tteck/Proxmox/assets/8570961/6903f285-0f62-4f89-90a7-bebd26343e9c)

**Comment by tteck** [comment, marked as answer]
There is no update path provided. It only does a `apt update && apt upgrade` (I'll fix that)
https://github.com/tteck/Proxmox/blob/6dcc29afa7b5697741bd1fc8d3bf404730eef43a/ct/prowlarr.sh#L55-L63

Nowhere does it state that an update path is available.

![Screenshot 2024-04-01 3 57 20 PM](https://github.com/tteck/Proxmox/assets/21342670/38ee5e7f-c956-4a0c-b229-986a42fbd935)

**Reply by Hoosier-IT** [reply]
Apologies. I took it for granted perhaps it was part of it, similar to other scripts.

**Comment by tteck** [comment]
Here is a test to try to update. Backup first.
```
systemctl stop prowlarr
wget --content-disposition 'https://prowlarr.servarr.com/v1/update/master/updatefile?os=linux&runtime=netcore&arch=x64'
tar -xzf Prowlarr.master.*.tar.gz
rm -rf /opt/Prowlarr
mv Prowlarr /opt
chmod 775 /opt/Prowlarr
systemctl start prowlarr
```

**Reply by Hoosier-IT** [reply]
Works like a charm! Thanks!

**Reply by bobbyrabbit** [reply]
This worked for me too :-)

**Comment by DrEVILish** [comment]
I've created an 'arr-update.sh' script that updates the *ARR series of apps.

```
function update_script() {
header_info
if [[ ! -d /var/lib/readarr/ ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
msg_info "Updating $APP LXC"

         RUN arr-update.sh $APP

msg_ok "Updated $APP LXC"
exit
}
```

https://github.com/DrEVILish/Proxmox/blob/main/misc/arr-update.sh

I'd like some input back on this.

**Comment by oOStroudyOo** [comment]
Prowlarr has a built-in updater.

Settings > General > Show Advanced > Updates > Choose your preferred settings.

**Reply by DrEVILish** [reply]
Correct it does but LXCHelper allows for an option to update ALL LXCs on Proxmox from one CLI command. 
Ensuring this "update" script does everything you expect it to do.
